HappyAnding/HappyAnding/Views/Components/MyShortcutCardListView.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  MyShortcutCardListView.swift
+//  HappyAnding
+//
+//  Created by KiWoong Hong on 2022/10/22.
+//
+
+import SwiftUI
+
+// MARK: 더미데이터를 위한 임시 구조체. 나중에 지울꺼
+/// PR날리고 머지되면 나중에 컴파일 에러날까봐 Shortcut에서 MyShortcut으로 이름 바꿈
+///  더미데이터 생성 머지되면 바로 지울예정!
```

Comment:
```suggestion
 // TODO: 더미데이터를 위한 임시 구조체. 더미데이터 머지 후 삭제 필요
 // PR날리고 머지되면 나중에 컴파일 에러날까봐 Shortcut에서 MyShortcut으로 이름 바꿈
``` 

TODO의 경우 추후 진행되는 사항에 작성한다고 합니다!
슬래시를 세개치는 경우는 문서화할 때 필요한 것으로 매개변수나 함수 자체 설명할 때 하면 좋을 것 같네요~

Reply:
오호 넵 수정하도록 할께요!